components/layouts/masonry.module.css

```diff
@@ -1,5 +1,5 @@
 .Container {
-  @apply columns-1 lg:columns-2 2xl:columns-3 gap-8 max-w-full;
+  @apply columns-1 lg:columns-2 gap-8 max-w-full;
```

Comment:
The rendered cheat sheet looks much more readable with this change! Thank you 😄 

Could we document why this change was done in either the PR description or comments for future reference?